test/config-test.js

```diff
@@ -24,6 +24,30 @@ suite('config-test', () => {
     });
   });
 
+  test('it assigns apiKey as storefrontAccessToken', () => {
```

Comment:
small thing about testing, I usually prefer to avoid variables in a case like this and do it more explicitly/directly. It makes it a little easier to quickly see what's going on. Here's the alternative:

```js
test('it assigns apiKey as storefrontAccessToken', () => {
  const config = new Config({
    domain: 'website.com',
    apiKey: 'i am actually a storefrontAccessToken'
  });

  assert.equal(config.storefrontAccessToken, 'i am actually a storefrontAccessToken');
});
```